### PR TITLE
Fix/update user

### PR DESCRIPTION
**Pull request by koutsilisk**
In this pr we fix:
- User update to handle roles updates

Info:
We use a low level update function to update models and it works perfectly from model fields. User roles it's an association and not a field thought so it can't handle that.

Quick fix:
Move the roles update to the ORM level that can handle it.

### CODE REVIEW

File: backend/project/tests/conftest.py [hunk 1]
```diff
@@ -57,21 +57,21 @@ def group(db_session):
...
-def group_2(db_session):
+def role(db_session, group):
     session = db_session
-    group_2 = Group(name="airbus_2", description="airbus 2 group")
-    session.add(group_2)
+    role = Role(name="user", description="user role", group=group)
+    session.add(role)
     session.commit()
-    yield group_2
+    yield role
```
Comment (Chichilele) [inline backend/project/tests/conftest.py:74]: just swapping?

File: backend/project/api/services/users.py [hunk 2]
```diff
@@ -111,6 +111,13 @@ def update(self, user_id: int, **kwargs) -> User:
         """
         updated_user = self.get_by_id(user_id)
 
+        # we do this here because of the assotiation,
+        # _update_model is too low level and can't update
+        # the assotiation between user and roles
+        if "roles" in kwargs:
+            updated_user.roles = kwargs["roles"]
```
Comment (Chichilele) [inline backend/project/api/services/users.py:118]: Is that enough without `session` `add` and `commit`?

**Review by Chichilele — APPROVED** [review]

File: backend/project/tests/conftest.py (hunk 1 above)
Comment (koutsilisk) [reply backend/project/tests/conftest.py:74]: yes, ocd :P

File: backend/project/api/services/users.py (hunk 2 above)
Comment (koutsilisk) [reply backend/project/api/services/users.py:118]: also if we commit here, and the the update raises an error the  rollback wont apply for the roles

**Pull request merged**